package/MDAnalysis/analysis/pca.py

```diff
@@ -0,0 +1,347 @@
+        to 1.
+    pca_space : array (n_frames, n_components)
+        After running :method:`pca.transform(atomgroup)` the projection of the
+        positions onto the principal components will exist here.
+    mean_atoms: MDAnalyis atomgroup
+        After running :method:`PCA.run()`, the mean position of all the atoms
+        used for the creation of the covariance matrix will exist here.
+
+    Methods
+    -------
+    transform(atomgroup, n_components=None)
+        Take an atomgroup or universe with the same number of atoms as was
+        used for the calculation in :method:`PCA.run()` and project it onto the
+        principal components.
+    """
+
+    def __init__(self, atomgroup, select='all', align=False, mean=None,
+                 n_components=None, **kwargs):
+        """
+        Parameters
+        ----------
+        atomgroup: MDAnalysis atomgroup
+            AtomGroup to be used for PCA.
+        select: string, optional
+            A valid selection statement for choosing a subset of atoms from
+            the atomgroup.
+        align: boolean, optional
+            If True, the trajectory will be aligned to a reference
+            structure.
+        mean: MDAnalysis atomgroup, optional
+            An optional reference structure to be used as the mean of the
+            covariance matrix.
+        n_components : int, optional
+            The number of principal components to be saved, default saves
+            all principal components, Default: None
+        start : int, optional
+            First frame of trajectory to use for generation
+            of covariance matrix, Default: None
+        stop : int, optional
+            Last frame of trajectory to use for generation
+            of covariance matrix, Default: None
+        step : int, optional
+            Step between frames of trajectory to use for generation
+            of covariance matrix, Default: None
+        """
+        super(PCA, self).__init__(atomgroup.universe.trajectory,
+                                  **kwargs)
+        if self.n_frames == 1:
+            raise ValueError('No covariance information can be gathered from a'
+                             'single trajectory frame.\n')
+
+        self._u = atomgroup.universe
+
+        if self._quiet:
+            logging.disable(logging.WARN)
+        # for transform function
+        self.align = align
+        # access 0th index
+        self._u.trajectory[0]
+        # reference will be 0th index
+        self._reference = self._u.select_atoms(select)
+        self._atoms = self._u.select_atoms(select)
+        self.n_components = n_components
+        self._n_atoms = self._atoms.n_atoms
+        self._calculated = False
+
+        if mean is None:
+            warning.warn('In order to demean to generate the covariance matrix'
+                         ' the frames have to be iterated over twice. To avoid'
```

Comment:
I normally let the white space at the end of the previous line. This is also what we do in most line breaking strings in mdanalysis (afaik that is some kind of unspoken standard between languages)